packages/twenty-front/src/modules/workflow/workflow-trigger/components/WorkflowEditTriggerCronForm.tsx

```diff
@@ -0,0 +1,305 @@
+        iconColor={theme.font.color.tertiary}
+        initialTitle={headerTitle}
+        headerType={headerType}
+      />
+      <WorkflowStepBody>
+        <Select
+          dropdownId="workflow-edit-cron-trigger-interval"
+          label="Trigger interval"
+          fullWidth
+          disabled={triggerOptions.readonly}
+          value={trigger.settings.type}
+          options={CRON_TRIGGER_INTERVAL_OPTIONS}
+          onChange={(newTriggerType) => {
+            if (triggerOptions.readonly === true) {
+              return;
+            }
+
+            triggerOptions.onTriggerUpdate({
+              ...trigger,
+              settings: getCronTriggerDefaultSettings(newTriggerType),
+            });
+          }}
+          withSearchInput
+        />
+        {trigger.settings.type === 'CUSTOM' && (
+          <FormTextFieldInput
+            label="Expression"
+            placeholder="0 0 */1 * * *"
+            error={errorMessages[0]}
+            hint="Format: [Second] [Minute] [Hour] [Day of Month] [Month] [Day of Week]"
+            readonly={triggerOptions.readonly}
+            defaultValue={trigger.settings.pattern}
+            onPersist={(newPattern: string) => {
+              if (triggerOptions.readonly === true) {
+                return;
+              }
+
+              const cronValidator = cron(newPattern, {
+                override: {
+                  useSeconds: true,
+                },
+              });
+
+              if (cronValidator.isError()) {
+                setErrorMessages({
+                  0: `Invalid cron pattern, ${cronValidator
+                    .getError()[0]
+                    .replace(/\. \(Input cron:.*$/, '')}`,
+                });
+                return;
+              }
+
+              setErrorMessages({});
+
+              triggerOptions.onTriggerUpdate({
+                ...trigger,
+                settings: {
+                  ...trigger.settings,
+                  type: 'CUSTOM',
+                  pattern: newPattern,
+                },
+              });
+            }}
+          />
+        )}
+        {trigger.settings.type === 'HOURS' && (
+          <>
+            <FormNumberFieldInput
+              label="Hours Between Triggers"
+              error={errorMessages[0]}
+              defaultValue={trigger.settings.schedule.hour}
+              onPersist={(newHour) => {
+                if (triggerOptions.readonly === true) {
+                  return;
+                }
+
+                if (!isNumber(newHour)) {
+                  return;
+                }
+
+                if (newHour <= 0) {
+                  setErrorMessages((prev) => ({
+                    ...prev,
+                    0: `Invalid hour value '${newHour}'. Should be integer greater than 1`,
+                  }));
+                  return;
+                }
+
+                setErrorMessages((prev) => {
+                  delete prev[0];
+                  return prev;
```

Comment:
The `delete` instruction is not compatible with the "no mutation" rule of React as it will mutate the previous value.